index.html

```diff
@@ -279,6 +279,8 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 															<span>Vitis AI (Preview)</span></div>
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="XNNPACK">
 															<span>XNNPACK (Preview)</span></div>
+														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="CANN">
+															<span>CANN (Preview)</span></div>
```

Comment:
I thought you had moved this to alphabetical (should be after ArmNN (Preview)), but it reverted back to the end. Can you fix again?

Reply:
sorry, it's my mistake.
The new commit to fix it is ready, please take a look at it again, thanks a lot.